esphome/components/ble_presence/binary_sensor.py

```diff
@@ -51,6 +53,9 @@ async def to_code(config):
     await cg.register_component(var, config)
     await esp32_ble_tracker.register_ble_device(var, config)
 
+    if CONF_MIN_RSSI in config:
+        cg.add(var.set_minimum_rssi(config[CONF_MIN_RSSI].as_hex))
```

Comment:
`as_hex` here is a little bit useless
```suggestion
        cg.add(var.set_minimum_rssi(config[CONF_MIN_RSSI]))
```